.github/workflows/tox-pytest.yml

```diff
@@ -160,63 +163,139 @@ jobs:
           workload_identity_provider: "projects/345950277072/locations/global/workloadIdentityPools/gh-actions-pool/providers/gh-actions-provider"
           service_account: "tox-pytest-github-action@catalyst-cooperative-pudl.iam.gserviceaccount.com"
 
-      - name: Run integration tests, trying to use GCS cache if possible
+      - name: Run Fast ETL
         run: |
-          tox -e integration -- --gcs-cache-path=gs://zenodo-cache.catalyst.coop --durations 0
+          pudl_setup
+          ferc_to_sqlite src/pudl/package_data/settings/etl_fast.yml
```

Comment:
I suspect `ferc_to_sqlite` should also use the `--gcs-cache-path` argument so it doesn't attempt to download anything from Zenodo unnecessarily.

Reply:
To use the `conda` environment that you have create with `setup-micromamba` I think you may need to use the [`micromamba-shell` command](https://github.com/mamba-org/setup-micromamba#custom-micromamba-shell-wrapper) -- otherwise I think everything is just going to be running in the system python environment.  I suspect this is why you've had to manually install libsnappy-dev.  I suspect that you've `pip install`ed the PUDL package in the system python environment.

Reply:
The new setup step claims to hack the default shell rc files to ensure that the new environment is run automatically and that indeed seems to be the case here. Win win.

Reply:
It seems that for the full ETL run, having both ferc_to_sqlite and pudl_etl part of a single command would be AFAIK nice simplification. That is not critical, but would be nice.

It does seem that in order to run ETL here, we effectively need to replicate the runtime environment we have in docker quite a bit (installing various dependencies, running alembic upgrade head), which might be another reason why running from docker image built once might be preferrable.